velox/common/process/TraceContext.cpp

```diff
@@ -18,6 +18,7 @@
 
 #include "velox/common/process/TraceHistory.h"
 
+#include <iostream>
```

Comment:
Is this required ? Can we get rid of it ?

Reply:
removed.